src/components/inputs/CheckboxSet/CheckboxSet.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+import Label from '../Label/Label';
+import Checkbox from '../Checkbox/Checkbox';
+import {uniqueId} from '../../../util';
+
+/**
+ * CheckboxSet component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {object[]} props.choices  The choices
```

Comment:
we also need to document the structure of a choice, otherwise how will a developer know?

```js
/**
  ...
  * @param {string} props.choices[].value  The value of the choice
  * @param {string} props.choices[].title  The label of the choice